modules/virt-vm-custom-scheduler.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-schedule-vms.adoc      
+
+:_content-type: PROCEDURE                                             
+[id="virt-vm-custom-scheduler_{context}"]                                   
+= Scheduling virtual machines with a custom scheduler
+
+You can use a custom scheduler to schedule a virtual machine (VM) on a node.
+
+.Prerequisites
+* A secondary scheduler is configured for your cluster.
+
+.Procedure
+
+* Add the custom scheduler to the VM configuration by editing the `VirtualMachine` manifest. For example:
++
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: vm-fedora
+spec:
+  running: true
+  template:
+    spec:
+      schedulerName: my-scheduler <1>
+      domain:
+        devices:
+          disks:
+            - name: containerdisk
+              disk:
+                bus: virtio
+# ...
+----
+<1> The name of the custom scheduler. If the `schedulerName` value does not match an existing scheduler, the `virt-launcher` pod stays in `Pending` state until the specified scheduler is found.
+
+
+.Verification
+* Verify that the VM is using the custom scheduler specified in the `VirtualMachine` manifest by checking the `virt-launcher` pod events:
```

Comment:
Would you say that these verification steps are sequential, for example, you do the first bullet point command, and then the second bullet point command? or would you say that they're two separate/independent steps?

If it's the first, I'd say that the verification procedure here should be numbered and not bulleted. If it's the latter, it's probably OK as-is. We can do a sequential/numbered procedure in a verification procedure.

Reply:
@stevsmit This is a single step procedure, so I have used `*` instead of `.` per our guidelines. The two subprocedure steps are sequential. I have changed the formatting to indicate that.